src/js/src/components/SomethingWrongControl.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import Control from 'react-leaflet-control';
+import { func, oneOf } from 'prop-types';
+
+import { controlPositionsEnum, osmUrl } from '../constants';
+
+function navigateToEditor({ zoom, center }) {
+    window.open(`${osmUrl}${zoom}/${center.lat}/${center.lng}`);
+}
+
+export default function SomethingWrongControl({
+    position,
+    getZoomAndCenter,
+}) {
+    return (
+        <Control position={position}>
+            <div className="something-wrong">
+                Something look wrong?&nbsp;
+                <button
+                    onClick={() => navigateToEditor(getZoomAndCenter())}
```

Comment:
This works well, so it's probably fine to leave as is. However, it occurred to me that all of the `window.open` logic could potentially be moved up into/in place of the `getCenterAndZoom` method bound to the class above.

Reply:
Seems low-consequence, but I like having the logic of the action be contained in the component and the map just providing the context it needs to do it.  I'm going to keep this as-is.